src/components/Tree/TreeItem.spec.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import sinon from 'sinon';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import TreeItem  from './TreeItem';
+
+describe('<TreeItem />', () => {
+  let props;
+  beforeEach(() => {
+    props = {
+      className: 'js-tree-item',
+      indent: 2,
+      label: (item) => <>Item - {item}</>,
+      option: {
+        item: 'X',
+        children: [
+          {
+            item: 'Y',
+            children: undefined,
+            selected: false
+          }
+        ],
+        selected: false,
+      },
+      selectable: true,
+      updateOption: sinon.stub()
+    };
+  });
+
+  afterEach(() => sinon.restore());
+
+  it('should render correctly', () => {
+    const wrapper = shallow(<TreeItem {...props} />);
+
+    assert(wrapper);
```

Comment:
Also, instead of assert we are using the jest `expect` now. It should be available with no imports.